src/app/pagenotfound/pagenotfound.component.scss

```diff
@@ -0,0 +1,44 @@
+@use '@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+.about-dockstore {
```

Comment:
Slightly wary of all this component-specific styling, but it is a unique page unlike any other, so... I always defer to Kathy on this; I should get more up to speed.